src/plugins/pendo/components/OptInModal/OptInModalBody.tsx

```diff
@@ -51,13 +58,8 @@ export const OptInModalBody: React.FC<OptInModalProps> = ({ onContinue }) => {
     onContinue()
   }
 
-  // If the user has already consented, we don't need to ask them again
-  if (typeof consent === 'boolean') {
-    return null
-  }
-
   return enabled ? (
-    <>
+    <LoadingBody isLoaded={typeof consent !== 'boolean'}>
```

Comment:
Since it will never be false but true or undefined, truthiness check should work the exact same way
```suggestion
    <LoadingBody isLoaded={!consent}>
```